content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -401,7 +393,7 @@ To make sure you catch these new errors you can adjust your code accordingly.
 
 #### Example of catching errors
 
-The following example uses the below blog schema with a 1-m relationship between `Post` and `User` and sets a [`Restrict`](#restrict) referential actions on the `author` field.
+The following example uses the below blog schema with a one-to-many relationship between `Post` and `User` and sets a [`Restrict`](#restrict) referential actions on the `author` field.
 
```

Comment:
The very last section in the doc is the one with the odd title/heading. Please see [this possible rewrite](https://www.notion.so/prismaio/Lucy-Tana-68842761500742819698c21212df965e) of the title and section.

Reply:
Thanks, done